packages/lodestar-utils/src/logger/winston.ts

```diff
@@ -80,7 +80,7 @@ export class WinstonLogger implements ILogger {
   }
 
   profile(message: string, option?: {level: string; message: string}): void {
-    this.winston.profile(message, option);
+    this.winston.profile(message, {level: LogLevel.verbose, ...option} as LogEntry);
```

Comment:
I seem to remember that this winston didn't respect the log level here. Haven't tested this out, so not sure if thats still the case.